### ISSUE TITLE
Missing browser prerequisite information

### ISSUE THREAD

**Issue by philwinder** [gptscript-ai/answers-from-the-internet#4]
OS: Mac 14.1.2
gptscript version v0.9.2+0f32a9d8

Expected: Running `gptscript github.com/gptscript-ai/answers-from-the-internet '{"question":"Who was Oliver Cromwell?"}'` should work

Actual: 

```
Error: No supported browsers (Chrome, Edge, Firefox) are installed. Error: browserType.launch: Chromium distribution 'chrome' is not found at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
Run "npx playwright install chrome",Error: browserType.launch: Chromium distribution 'msedge' is not found at /Applications/Microsoft Edge.app/Contents/MacOS/Microsoft Edge
Run "npx playwright install msedge",Error: browserType.launch: Executable doesn't exist at /Users/phil/Library/Caches/ms-playwright/firefox-1449/firefox/Nightly.app/Contents/MacOS/firefox
```

The issue here is that Brave is my default browser. I actually tried downloading firefox and that didn't work so I went down an npm path. But as it turns out if you look above it's looking for some weird nightly location. I don't know where that comes from.

So I just installed Chrome (yuck!) and then everything worked.

So basically, the question is, what's the prerequisites for this? Can this tool somehow manage it's own browser dependency internally so people don't have to muck around getting the global browser working? Or is it just a documentation issue?

p.s. I also messed around with npm since I thought that was the problem but it was not. I think I undid everything.
